source/jormungandr/jormungandr/autocomplete/geocodejson.py

```diff
@@ -206,7 +206,10 @@ def call_bragi(self, url, method, **kwargs):
     def _check_response(cls, response, uri):
         if response is None:
             raise GeocodeJsonError('impossible to access autocomplete service')
-        if response.status_code == 404:
+        if response.status_code in (400, 404):
```

Comment:
It feels weird to do it only for 400 and 404. Why not extending to the entire error codes 4XX-5XX?

Reply:
how about `response.status_code > 400` ?

Reply:
At this stage we just want to include 400 to use fallback kraken autocomplete.